test/internal/pbxproj_partials/write_files_and_groups_tests.bzl

```diff
@@ -64,8 +68,16 @@ def _write_files_and_groups_test_impl(ctx):
         struct(
             path = path,
             is_source = False,
+            owner = Label(values[0]),
```

Comment:
Instead of a label, which possibly does some odd resolution, I would create a struct that has the fields that we look at (and if we don't, a simple string).

Reply:
Ah good to know! fixed